packages/application-components/src/components/internals/for-docs/example-wrapper/example-wrapper.js

```diff
@@ -19,6 +19,7 @@ import { InfoDialog } from '@commercetools-frontend/application-components';
 const PreviewContainer = styled.div`
   position: relative;
   width: 100%;
+  overflow: hidden;
```

Comment:
What's this for?

Reply:
It's for the shadows around the portal not to "bleed" out of the PreviewContainer.
![image](https://user-images.githubusercontent.com/2941328/59431908-4e781d80-8de6-11e9-86d9-6f6394799c5a.png)
Design asked about this, but in practice, those shadows won't show on the edge of the pages, only the ones on the left. So I decided to clip them from the PreviewContainer to make it clearer how it will appear on a page.

Reply:
Ah ok, makes sense 👌